examples/firmware_bundle.rb

```diff
@@ -0,0 +1,21 @@
+# (c) Copyright 2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+my_client = {
+  url: '',
+  user: '',
+  password: ''
+}
+
+oneview_firmware_bundle 'firmware_bundle_name' do
+  file_path '/bundles/firmware_bundle_name'
```

Comment:
Put  a more meaningful name. It seems it's the path to a folder. Put something like `firmware_bundle.spp` for example.